db/seeds.rb

```diff
@@ -5,7 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-
+User.all.delete_all
+Band.all.delete_all
```

Comment:
the seeds file should only be run after the start of the project, so no need for these deletes